src/invocation/InvocationService.ts

```diff
@@ -8,19 +8,27 @@ import Address = require('../Address');
 import ExceptionCodec = require('../codec/ExceptionCodec');
 import {BitsUtil} from '../BitsUtil';
 
-export interface Invocation {
+var EXCEPTION_MESSAGE_TYPE = 109;
+var INVOCATION_TIMEOUT = 120000;
+var INVOCATION_RETRY_DELAY = 1000;
+
+class Invocation {
```

Comment:
If we do not `export Invocation` no other class can create invocation by itself and call `InvocationService.invoke` directly. Do we want that? I think `Invocation` should be exported.